AjobK/Athena
# 298 profile card revamp

closes #298 

<br />

 **What is it supposed to do**  
Change the banner and profile card on the profile page to align with the designs on Zeplin.io

<br />

 **How can we test your branch**  
Open the profile page and compare this with the designs on Zeplin. Try updating your avatar/banner by hovering over them and selecting new images. Try following someone/adding a post and see if the values at the profile card update correctly. Edit your bio and see if this saves correctly.

<br />

 **Additional notes**  
Things to keep in mind regarding the PR that don't fit in the previous two headings. 

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**

---

**PR Rules**  
*1. Add relevant labels.*  
*2. Select at least two reviewers.*  
*3. Assign all reviewers allowed to merge the branch, including yourself.*  
*4. If relevant to the sprint, put related issue in 'Awaiting Approval' column on project board.*  
*5. Add to the corresponding [milestone](https://github.com/AjobK/Seaqull/milestones).*

<br />

**Reviewer info**  
*1. You can use aliases to refer to specific PR checklist items. For example 'Hey @JohnDoe, for this line please check 3.a and 1.d.'*  
*2. Where possible, please use the suggestions feature in github so the author has a better idea of the solution you had in mind.*   

<br />

> This form should be filled in by the author of this PR.
